catalog/core/catalog-core-validator/src/main/java/ddf/catalog/validation/impl/validator/MatchAnyValidator.java

```diff
@@ -21,16 +21,28 @@
 import ddf.catalog.validation.violation.ValidationViolation;
 import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
 import java.util.List;
 import java.util.Optional;
 import java.util.Set;
+import java.util.stream.Collectors;
 import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
 
 public class MatchAnyValidator implements AttributeValidator {
   private final List<AttributeValidator> validators;
 
   public MatchAnyValidator(List<AttributeValidator> validators) {
-    this.validators = validators;
+    if (validators == null) {
```

Comment:
✏️ Consider:
```
    this.validators =
        Optional.ofNullable(validators).orElse(Collections.emptyList()).stream()
            .sorted(Comparator.comparingInt(Object::hashCode))
            .collect(Collectors.toList());```

Reply:
I think we will go with what we have for now, at least to keep it same with 2.19.x version of DDF.